This PR will add the following changes if applied:

jaredmorgs opened the pull request:
- all aspects of algolia plug-in configuration using step-by-step instructions.
- made the book.json syntax match the current gitbook formatting.
- how to add an environment variable to a book through the editor.

jaredmorgs committed:
This PR will add the following changes if applied:

- all aspects of algolia plug-in configuration using step-by-step instructions.
- made the book.json syntax match the current gitbook formatting.
- how to add an environment variable to a book through the editor.

redox commented on README.md:
```diff
@@ -6,7 +6,23 @@ This plugins requires gitbook `>=3.0.0`.
+    * **Index ID**, which is the name of the index you set up in Algolia web.
+    * **Application ID**, which is your unique application ID for API identification. 
+    * **Search-only API Key**, which is the unique ID for search queries.
+    * **Is your algoria account free**, which you leave as `true` if you have what Algoria call a "Hacker Account".
```
`s/Algoria/Algolia/` small typo :)

jaredmorgs replied:
Good catch @redox . I've corrected this.

jaredmorgs committed:
Update README.md

Minor fix to typo

redox commented on README.md:
```diff
@@ -6,7 +6,23 @@ This plugins requires gitbook `>=3.0.0`.
+1. Open Settings > Plugins Store interface in the Book Editor.
+2. Install the `algolia` plugin.
+3. Specify the following values in the modal for your book:
+    * **Index ID**, which is the name of the index you set up in Algolia web.
```
I would go for `Index Name` instead of `Index ID`, but minor wording issue.

jaredmorgs committed:
Made modal labels parallel 

Changed the names of the modal labels to match the (current) modal labels. Better to have them exact matches.

Soreine commented:
Thank you :)
